src/randomInt.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Generate an inclusive random integer between `from` and `to`.
+ *
+ * @param from - The minimum value.
+ * @param to - The maximum value.
+ * @returns The random integer.
+ * @signature
+ *   R.randomInt(from, to)
+ * @example
+ *   R.randomInt(1, 10) // => 5
+ *   R.randomInt(1n, 10n) // => 7n
+ * @dataFirst
+ * @category Number
+ */
+export function randomInt<T extends bigint | number>(
+  from: T,
+  to: Widen<T>,
+): Widen<T> {
+  if (to <= from) {
+    throw new RangeError(`to(${to}) should be greater than from(${from})`);
+  }
+
+  if (typeof from === "bigint" && typeof to === "bigint") {
+    return BigInt(
+      Math.floor(Number(to - from + BigInt(1)) * Math.random() + Number(from)),
+    ) as never;
+  }
```

Comment:
This is not an accurate random number generator for `bigint`s because `Math.random` generates just shy of 2^62 different values. If our range of `bigint` is bigger than that (which it might be if the user is already using `bigint`) then this calculation will not result in an unbiased distribution of the values, as some of them will never be returned.

To make things simple I suggest removing the support for `bigint` from this function. We can consider creating a separate `randomBigInt` utility, but i'm not sure the built-in libraries in ECMAScript provide a way to generate it (it's solvable using the crypto module, but I don't think that's standard enough).

Reply:
Hmm, makes sense

Removed